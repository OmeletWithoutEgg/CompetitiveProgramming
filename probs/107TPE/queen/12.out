12350686.53 -78831183.59
