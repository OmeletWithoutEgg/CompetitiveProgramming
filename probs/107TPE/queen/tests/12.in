224
101637065 -57591160
-37431565 31185653
63430106 -18112819
-49176919 17641597
-64082059 83584879
-76094445 72237109
-59299840 84803203
-78834112 66260619
90763625 -63873899
-67137494 18360471
100062388 -59041045
-80003974 62496759
60799173 -32781438
78630285 -63025758
-81126748 43574085
-76955244 70631868
91511043 -63678302
61036062 -36773145
79053857 -63185883
-63941044 83641862
-74542275 25656805
65031237 -50148160
-66494851 82395338
-81279535 55530695
110531923 -40169754
-34977000 62111819
-39433151 27367090
-69520311 80273185
60886633 -29673396
88997607 -64219069
-73464142 76177115
-63470363 16544731
60833656 -34096377
94693256 -62498471
77816833 -2053995
63037593 -19164670
-62653888 16266109
-80254389 61497233
73983276 -4199238
108026987 -16603644
103858419 -55093667
-80695994 40925355
61654907 -40659682
-80914467 58214446
-34857039 61627040
-75209620 26662017
-76343701 28575233
67594051 -54261626
110569258 -39978097
-55046686 15658166
66198968 -12547767
82365066 -64097165
-36225197 66202575
110979073 -37387111
74849570 -3628188
70503048 -57644110
-79936157 37615170
110452343 -40562408
-33966427 43478561
-61286668 15895362
63553737 -17800759
-45470807 20168425
104639016 -10681957
88039978 -64338931
-35060630 62436991
111207069 -29777925
-33990787 57064965
-80420283 39590670
62440077 -43750873
106444567 -51261713
-38944284 28203181
-74818566 74303940
72186808 -5565555
-47107917 18913703
-33947680 43617642
-36017880 34754063
63009277 -19244544
-37129311 68496312
97332818 -61054594
110939376 -27047325
76559194 -62090530
-77316204 69896589
68772400 -8981700
-37249234 31594738
-72503065 77342901
69644984 -56753637
64841725 -49781335
91884124 -63569402
68812111 -8934494
-77971693 31919210
109286122 -19851552
108518971 -46977605
-41042228 75418957
-41045187 24944165
60922780 -29175900
61028931 -36707279
-47177415 18865503
111289506 -32520463
-71267681 21694720
-49993612 83139661
110497776 -40340764
76521816 -62071240
-56502081 15503139
-67895457 18860405
87029793 -64414881
-35683032 35770077
72584201 -5240847
95611520 -62047147
67414084 -54014315
-34881702 38638576
-41853257 76476392
-67313156 18472265
-34092097 42614408
61470513 -25014444
-41839664 23907709
91023045 -63809396
-42988719 77805720
-57623251 15470933
-60429331 84644535
-58721332 84854457
60802398 -33027813
88077617 -64335103
-33660480 54011309
62127634 -42644562
-45616213 20047354
-36533748 33335242
108115026 -16803341
62604719 -20455812
-52848241 16139050
103139594 -55967797
-80437657 39669328
-35357357 63518113
61755470 -41123171
-81431869 46516757
-60794321 84576556
-37224107 31652114
-49532478 82911719
-80582242 40350756
110389741 -40857684
61455895 -25093668
107055445 -50148555
-81412106 46257261
88551792 -462410
60966508 -36080148
-65247610 17305351
-57461436 15470912
77445972 -2220472
70868180 -57999601
95186884 -62262359
-36922452 68006035
-57357119 15471733
63966281 -47926990
109799698 -43225443
-76026684 28011583
60902680 -29442010
-48647925 17935687
-58668722 84858115
-67977277 18917143
-34238810 41721339
-68369916 19197139
-35543815 36219160
-81540968 51776881
-68867832 80796155
94444525 -62611820
66923187 -53312206
77962391 -1990900
61363456 -25617313
-70326114 79572799
81034543 -941220
81839156 -751526
103493033 -9196491
73827511 -4307724
-58196673 15483688
101488205 -7004582
109244739 -19728321
-46968549 19011561
102756174 -8336219
97195401 -61141194
78795960 -63089610
-72373663 22876488
84583739 -64385719
91654831 -1105878
-81130612 43602935
-36610211 67229399
96689988 -3294629
76924346 -62274354
94013496 -62799478
-73094179 23726567
-73629227 24403120
-50786702 83496898
-55912482 15550472
-38437918 29129105
-70665114 21107977
-33933985 43721208
64150592 -16401364
-70076434 20570678
78598741 -1729684
-50405809 83330754
66232398 -52249212
-78373715 67473974
81952276 -64015510
111004452 -37179122
82071251 -64039957
67795503 -10210713
91804904 -63593159
-33541160 48532423
-79426656 64503548
63663433 -17530758
-77955114 31880552
72665874 -5175753
109764721 -21394934
102103878 -57113990
-45190787 79960018
-72344531 22843508
84515480 -64380579
111080156 -28247723
103273259 -55810413
-53828951 84479825
88158882 -416514
69777368 -56896290
-77685491 69099945
-67283773 18453396
-65220224 83075182
64027264 -16677287
