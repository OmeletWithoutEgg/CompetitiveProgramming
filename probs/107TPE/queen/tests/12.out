15930253.56 7430470.57
